include/net/icalendar_parser.rst

```diff
@@ -0,0 +1,38 @@
+.. _icalendar_parser_readme:
+
+iCalendar parser
+################
+
+The iCalendar parser library can be used to parse a data stream in `iCalendar`_ format, which is a data format for representing and exchanging calendaring and scheduling information.
+The library parses the calendaring information and returns parsed calendar events.
+
+The library first detects the beginning of the calendar object by locating the delimiter ``BEGIN:VCALENDAR``.
+It then parses the following calendar content fragment by fragment.
+For each calendar component that is parsed, the library sends a parsed event (:c:type:`ical_parser_evt`) to the application.
+
+Supported features
+******************
+
+The library supports iCalendar version 2.0 as defined in `RFC5545 <iCalendar_>`_.
+
+The iCalendar parser library supports parsing the following calendar components:
+
+* VEVENT
+
+It supports parsing the following calendar properties:
+
+* DESCRIPTION
```

Comment:
I changed the order for these properties to be alphabetic. Is that okay, or should they be ordered differently?

Reply:
It is totally ok. The order in RFC is also in alphabetic.